_posts/programme/2021-03-24-finale.md

```diff
@@ -41,6 +41,9 @@ All times in UTC.
 - **16:05**: The International Council of RSE Associations
 
   Stephan Druskat, _German Aerospace Center (DLR), Humboldt-Universität zu Berlin, Friedrich Schiller University Jena_
+
+  > **Abstract:** The UK RSE association was founded in [2013](https://society-rse.org/about/history/), and since then, a number of national RSE associations and initiatives have formed: there are now groups in [Germany](https://de-rse.org), [the Netherlands](https://nl-rse.org/), [Africa](https://rsse.africa/), [Australia/New Zealand](https://rse-aunz.github.io), [the Nordics](https://nordic-rse.org/), [Belgium](https://www.be-rse.org/), and the [United States](https://us-rse.org/), along with the [Society of Research Software Engineering](https://society-rse.org) in the UK. These groups informally talk and collaborate by a variety of means, be it serving on committees for each others' national conferences, running a concerted RSE survey, or this very series of online research software events. The recently founded [International Council of RSE Associations](https://researchsoftware.org/2021/01/27/introducing-the-international-council-of-RSE-associations.html) provides a formal open forum for national and multinational RSE associations to talk and coordinate regularly, and thus sustain this international collaboration. In this talk, I will briefly introduce the Council, and its aims and setup.
```

Comment:
```suggestion
  > **Abstract:** The UK RSE association was founded in [2013](https://society-rse.org/about/history/), and since then, a number of formal national RSE associations and initiatives have formed: there are now groups in [Germany](https://de-rse.org), [the Netherlands](https://nl-rse.org/), [Africa](https://rsse.africa/), [Australia/New Zealand](https://rse-aunz.github.io), [the Nordics](https://nordic-rse.org/), [Belgium](https://www.be-rse.org/), and the [United States](https://us-rse.org/), along with the [Society of Research Software Engineering](https://society-rse.org) in the UK. These groups informally talk and collaborate by a variety of means, be it serving on committees for each others' national conferences, running a concerted RSE survey, or this very series of online research software events. The recently founded [International Council of RSE Associations](https://researchsoftware.org/2021/01/27/introducing-the-international-council-of-RSE-associations.html) provides a formal open forum for established national and multinational RSE associations to talk and coordinate regularly, and thus sustain this international collaboration. In this talk, I will briefly introduce the Council, and its aims and setup.
```

Reply:
Thanks @cosden!